workflow-ui/core-android/src/main/java/com/squareup/workflow1/ui/WorkflowViewStub.kt

```diff
@@ -94,6 +94,7 @@ class WorkflowViewStub @JvmOverloads constructor(
     inflatedId = attrs.getResourceId(R.styleable.WorkflowViewStub_inflatedId, NO_ID)
     attrs.recycle()
 
+    visibility = GONE
```

Comment:
Makes WorkflowViewStub behave similar to ViewStub in the scenario specified in https://github.com/square/workflow-kotlin/issues/170